scss/_variables.scss

```diff
@@ -788,8 +788,12 @@ $mark-bg-dark:                $white !default; // Boosted mod
 // scss-docs-start table-variables
 $table-cell-padding-y:      .875rem !default;
 $table-cell-padding-x:      $spacer * .5 !default;
+$table-cell-icon-margin-top:        -.688rem !default; // Boosted mod
```

Comment:
All the right variables should be aligned

Reply:
Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1674/commits/254e71b689049a14aa15425a2c5ef43386761195